CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+v1.2.0
+------
+* Roll back the use of psycopg2 connection pools. (See [tilequeue/#149](https://github.com/tilezen/tilequeue/pull/149) and [#62](https://github.com/tilezen/tileserver/pull/62))
+* Try increasing the maximum connection count on connection pools. (See [#61](https://github.com/tilezen/tileserver/pull/61))
```

Comment:
If we're not releasing this change by tagging 1.2.0, let's not mention it in the changelog.